app/controllers/v3/organization_quotas_controller.rb

```diff
@@ -94,7 +90,7 @@ def apply_to_organizations
     organization_quota = QuotaDefinition.first(guid: hashed_params[:guid])
     resource_not_found!(:organization_quota) unless organization_quota
 
-    OrganizationQuotaApply.new.apply(organization_quota, message)
+    OrganizationQuotaApply.new.apply(organization_quota, message, **presenter_args)
```

Comment:
Not needed as only admins can reach this...

Reply:
Yep, I was thinking that one might be unnecessary. Reverting this.